data/hai/hai reveal 5 philosophers.txt

```diff
@@ -1011,84 +1011,129 @@ mission "Hai Reveal [B13] Freedom"
-			`	The squads move room to room, neutralizing all opposition with relative ease until their opponents switch tactics. The pirates begin using electromagnetic pulse devices to shut down the suits (and all other electronics nearby), leaving the targeted operatives helpless for up to ten seconds. Here some of General Choot'k's Unfettered show their strength, darting between operatives and bounding off walls with pulse rifles and long knives. They strike rapidly and withdraw as the operatives suits re-engage, preventing the pirates from coordinating an opportunistic counter-assault.`
-			`	After a bloody struggle, forty-three surviving operatives win the fight, clearing the way to a large blast door. Only a couple dozen operatives are critically or fatally wounded, but the pirates were very good at booby trapping things or utilizing dormant machinery to pin, crush, or disable operatives or their suits. A great many of them will be spending time in a hospital. The Unfettered fared comparatively better, having mostly acted as reserve and flanking forces; still, over a dozen of them will be taking their own tails home as trophies after a few groups of pirates used miner's cutting power-arm extensions to great effect.`
-			`	Having worked through the spaceport and down the access tunnel to the settlement, the vast blast door that the force now stands before is twelve meters high and wide, and scanners show it to be six meters deep. It is designed to be closed from this side though, and has not been sealed very well against being reopened from this side. Before long it grinds part-way open, evidently sabotaged, and the operatives put on their goggles and pass through into the darkness of the next room.`
+			`General Choot'k cuts open the doors of a large hangar, and your ships land. The two hundred Syndicate operatives pour out wearing mechanized battle suits with armor-plated exoskeletons and weapons larger than a human could carry. They storm through the complex and you follow in their wake. Their guns melt away meter-thick walls in seconds, and most of their opposition lacks protection against the suits' stun gas bombs. The pirates outnumber the Syndicate's operatives twenty to one, but this huge advantage in equipment, combined with training and discipline, gives the operatives an overwhelming advantage in constrained spaces.`
```

Comment:
```suggestion
			`General Choot'k cuts open the doors of a large hangar, and your ships land. The two hundred Syndicate operatives pour out wearing mechanized battle suits with weapons larger than a human could carry. They storm through the complex and you follow in their wake. Their guns melt away meter-thick walls in seconds, and most of their opposition lacks protection against the suits' stun gas bombs. The pirates outnumber the Syndicate's operatives twenty to one, but this huge advantage in equipment, combined with training and discipline, gives the operatives an overwhelming advantage in constrained spaces.`
```
dont think we need to add that? I mean if its a mechanized battle suit it for sure has armor? idk though

Reply:
No, a mech suit can be unarmored. Besides, explicitly saying "armor-plated exoskeletons" emphasizes how durable they are.

Reply:
Sounds good to me, you can resolve this (always the case when I put a thumbs up)